src/js/components/ClickToSelect.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class ClickToSelect extends React.Component {
+  constructor() {
+    super();
+    this.selectAll = this.selectAll.bind(this);
+  }
+
+  selectAll() {
+    document.getSelection().selectAllChildren(this.refs.node);
```

Comment:
The support charts are full of mysterious `?` but I tested in latest Chrome, FF, Safari, Edge, and IE11 and it seemed to be functioning as advertised 👍